### ISSUE TITLE
using requirements, it seems the version request for guidata may be too demanding

### ISSUE THREAD

**Issue by stonebig** [PierreRaybaut/guiqwt#43]
````
  Could not find a version that satisfies the requirement guidata>=1.7.0 (from g
uiqwt->-r D:\WinpythonQt5\basedir34\requirements3.txt (line 2)) (from versions:
1.7.0rc1)
No matching distribution found for guidata>=1.7.0 (from guiqwt->-r D:\WinpythonQ
t5\basedir34\requirements3.txt (line 2))
````

Maybe the test should not be 1.7.0 but 1.6.99 , if 1.7.0rc1 is considered inferior to 1.7.0 by pip requirements algorithm.

**Comment by stonebig** [comment]
I tried with the source, but I see there is a wheel now... will retry with the wheel

**Comment by stonebig** [comment]
Aaaaaargh!
````
  Could not find a version that satisfies the requirement PIL>=1.1.6 (from guiqw
t->-r D:\WinpythonQt5\basedir34\requirements3.txt (line 2)) (from versions: )
No matching distribution found for PIL>=1.1.6 (from guiqwt->-r D:\WinpythonQt5\b
asedir34\requirements3.txt (line 2))
````

**Comment by PierreRaybaut** [comment]
Note that I've also released `guidata` 1.7.0 final to avoid this requirement issue. After all, I didn't intend to change anything significative in `guidata` between RC1 and final release.
